spring-batch-infrastructure/src/main/java/org/springframework/batch/item/file/FlatFileItemReader.java

```diff
@@ -189,7 +191,8 @@ protected T doRead() throws Exception {
 	/**
 	 * @return next line (skip comments).getCurrentResource
 	 */
-	private String readLine() {
+	@Nullable
+	protected String readLine() {
```

Comment:
The visibility of `readLine` has been changed from `private` to `protected`. Why is that?

Reply:
Fixed in 230614182667378924ec557913df75b72af0ddc5 as I believe this should not change.

Reply:
It was done for my internal tests. Thanks for the change.